code/client/munkilib/updatecheck/core.py

```diff
@@ -174,6 +174,16 @@ def check(client_id='', localmanifestpath=None):
                     localonlymanifest, 'managed_uninstalls') or []
                 for item in localonlyuninstalls:
                     analyze.process_removal(item, cataloglist, installinfo)
+
+                localonlyoptionals = manifestutils.get_manifest_value_for_key(localonlymanifest,
+                                                                              'optional_installs') or []
```

Comment:
This line is too long and doesn't match the style of this file. Please move `localonlymanifest, 'optional_installs') or []` to the second line.